app/views/posts/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for @post, html: {class:"new_post_form"} do |f| %>
+<div>
+  <%= f.label :title,:class => "label" %>
+  <%= f.text_field :title, :class => "title_box input" %>
```

Comment:
Is this `input` class something you apply to all `<input>` tags? If so, do you need the class? If not, perhaps there's a more semantic class name you could use.